draft-ietf-quic-recovery.md

```diff
@@ -549,9 +544,7 @@ expired, the sender SHOULD send ack-eliciting packets from other packet
 number spaces with in-flight data, coalescing packets if possible.
 
 When the PTO timer expires, and there is new or previously sent unacknowledged
-data, it MUST be sent.  Data that was previously sent with Initial encryption
-MUST be sent before Handshake data and data previously sent at Handshake
-encryption MUST be sent before any ApplicationData data.
```

Comment:
Why are you removing this text?

Reply:
This should have been removed in #3066, but was missed.

Reply:
Ah, right, of course.